build.sbt

```diff
@@ -60,6 +60,14 @@ Revolver.settings
 
 Revolver.enableDebugging(port = 5050, suspend = false)
 
-Defaults.itSettings
+def isIntegrationTest(name: String) = name contains "integrationtest"
 
-lazy val rawls = project.in(file(".")).configs(IntegrationTest)
\ No newline at end of file
+lazy val IntegrationTest = config ("it") extend (Test)
+
+lazy val rawls = project.in(file("."))
+  .configs(IntegrationTest)
+  .settings(inConfig(IntegrationTest)(Defaults.testTasks): _*)
+  .settings(
+    testOptions in Test := Seq(Tests.Filter(s => !isIntegrationTest(s))),
+    testOptions in IntegrationTest := Seq(Tests.Filter(s => isIntegrationTest(s)))
```

Comment:
Coveralls broke for Cromwell when we assigned to `testOptions in Test` rather than appending; this should maybe use `++=` instead.  

https://github.com/broadinstitute/cromwell/commit/7268d219627c27eec5a8036c0c26a36e6d9e1907